File: src/main/java/com/iflat/sm/bean/SbSettlement.java
package com.iflat.sm.bean;

import java.util.Date;

/**
 * Created by tyriv on 2016/3/21.
 */
public class SbSettlement {

    /**
     * 修改时需要查看流程状态，如果不为0，则不允许修改
     */
    private String id;
    private String projNo;
    private String projName;
    private Date month;
    private String deptName;
    private String team;
    private String attachment;
    private String comment;
    private int status;  // 流程状态 0申请 1已提交审批 2结束
    private String creatorAcc;
    private String creatorName;
    private Date createTime;

    public SbSettlement() {
        this.status = 0;
        this.createTime = new Date();
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getProjNo() {
        return projNo;
    }

    public void setProjNo(String projNo) {
        this.projNo = projNo;
    }

    public String getProjName() {
        return projName;
    }

    public void setProjName(String projName) {
        this.projName = projName;
    }

    public Date getMonth() {
        return month;
    }

    public void setMonth(Date month) {
        this.month = month;
    }

    public String getDeptName() {
        return deptName;
    }

    public void setDeptName(String deptName) {
        this.deptName = deptName;
    }

    public String getTeam() {
        return team;
    }

    public void setTeam(String team) {
        this.team = team;
    }

    public String getAttachment() {
        return attachment;
    }

    public void setAttachment(String attachment) {
        this.attachment = attachment;
    }

    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }

    public String getCreatorAcc() {
        return creatorAcc;
    }

    public void setCreatorAcc(String creatorAcc) {
        this.creatorAcc = creatorAcc;
    }

    public String getCreatorName() {
        return creatorName;
    }

    public void setCreatorName(String creatorName) {
        this.creatorName = creatorName;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }
}

File: web/app/model/sm/SbSettlement.js
Ext.define('iFlat.model.sm.SbSettlement', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'sbSettlement.id', mapping: 'id', type: 'string'},
        {name: 'sbSettlement.projNo', mapping: 'projNo', type: 'string'},
        {name: 'sbSettlement.projName', mapping: 'projName', type: 'string'},
        {name: 'sbSettlement.month', mapping: 'month', type: 'date'},
        {name: 'sbSettlement.deptName', mapping: 'deptName', type: 'string'},
        {name: 'sbSettlement.team', mapping: 'team', type: 'string'},
        {name: 'sbSettlement.attachment', mapping: 'attachment', type: 'string'},
        {name: 'sbSettlement.comment', mapping: 'comment', type: 'string'},
        {name: 'sbSettlement.status', mapping: 'status', type: 'string'},
        {name: 'sbSettlement.creatorAcc', mapping: 'creatorAcc', type: 'string'},
        {name: 'sbSettlement.creatorName', mapping: 'creatorName', type: 'string'},
        {name: 'sbSettlement.createTime', mapping: 'createTime', type: 'date'},
        {name: 'sbSettlement.amount', mapping: 'amount', type: 'number'},
        {name: 'sbSettlement.mgrScore', mapping: 'mgrScore', type: 'number', defaultValue: 100 },
        {name: 'sbSettlement.progressScore', mapping: 'progressScore', type: 'number', defaultValue: 100 },
        {name: 'sbSettlement.qualityScore', mapping: 'qualityScore', type: 'number', defaultValue: 100 },
        {name: 'sbSettlement.safetyScore', mapping: 'safetyScore', type: 'number', defaultValue: 100 },
        {name: 'sbSettlement.fineAmount', mapping: 'fineAmount', type: 'number', defaultValue: 0 },
        {name: 'sbSettlement.summaryAmount', mapping: 'summaryAmount', type: 'number'},
    ]
});